types/unix-time/.eslintrc.json

```diff
@@ -0,0 +1,6 @@
+{
+    "rules": {
+        "dt-header": "off",
+        "trim-file": "off"
+    }
```

Comment:
Please use template as suggested at https://github.com/DefinitelyTyped/DefinitelyTyped#create-a-new-package so you arent adding unnecessary files.